runtime/doc/options.txt

```diff
@@ -3552,6 +3552,31 @@ A jump table for the options with a short description can be found at |Q_op|.
 	  eob		EndOfBuffer		|hl-EndOfBuffer|
 	  lastline	NonText			|hl-NonText|
 
+						*'findexpr'* *'fexpr'*
```

Comment:
Might be worth mentioning that custom exprs can still support `path` if they choose?